app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesComposables.kt

```diff
@@ -416,17 +423,29 @@ fun PoweredByPocketHeader(
                 Text(
                     text = stringResource(R.string.pocket_stories_feature_title),
                     color = FirefoxTheme.colors.textPrimary,
-                    fontSize = 12.sp,
-                    lineHeight = 16.sp
+                    style = FirefoxTheme.typography.caption
                 )
 
-                ClickableSubstringLink(
```

Comment:
Do we need to keep `ClickableSubstringLink` after this?

Reply:
Left it in the project although not used now in the idea that it is meant to be reusable and may be useful later.
It makes more sense to use it when there is a string with only one part that is clickable - seeing that in this patch I'm using two strings and hardcoding a space character in between which to me seems uglier and riskier.
Can remove it in the idea that that is problematic for ui tests.